webapp/store/actions/socketActions.js

```diff
@@ -1,10 +1,12 @@
 import { CONNECT_SOCKET, DISCONNECT_SOCKET } from '../constants/sockets';
+import { bpanelClient } from '@bpanel/bpanel-utils';
 
 export function connectSocket() {
+  const client = bpanelClient();
```

Comment:
Maybe we should expose the functionality for retrieving the port and other data outside of creating a client and have the same function be consumed inside of `bpanelClient` because we don't really want or need to create a client in this case, we just want the config info

Reply:
I think we should expose the `options` function, maybe we could rename it so that it better describes what it does

Reply:
Do you think there's any tradeoff by not using the actual properties of the client? For what it's worth, `bpanelClient` doesn't actually create a new client but rather is a singleton that creates a client if there isn't one already, otherwise just returns the existing client (to avoid problems of creating new clients).